CMakeLists.txt

```diff
@@ -154,6 +155,23 @@ include_directories (/usr/local/include/)
 
 link_directories (/usr/local/lib)
 
+# External libraries
+
+# 1. openssl
+include_directories ("/usr/local/opt/openssl/include")
+set (OPENSSL_HINTS "/usr/local/opt/openssl/lib")
+find_library (LIBSSL libssl.a HINTS ${OPENSSL_HINTS})
+find_library (LIBCRYPTO libcrypto.a HINTS ${OPENSSL_HINTS})
+set (OPENSSL_LIBS ${LIBSSL} ${LIBCRYPTO})
+
+# 2. icu4c
+include_directories ("/usr/local/opt/icu4c/include")
+set (ICU_HINTS "/usr/local/opt/icu4c/lib")
+find_library (ICUI18N libicui18n.a HINTS ${ICU_HINTS})
+find_library (ICUUC libicuuc.a HINTS ${ICU_HINTS})
+find_library (ICUDATA libicudata.a HINTS ${ICU_HINTS})
+set (ICU_LIBS ${ICUI18N} ${ICUUC} ${ICUDATA})
+
```

Comment:
Well, this may be valid for any OS and distribution, possibly with small additions into HINTS. Generally, this is a common practice to access external libraries in CMake projects.